test-data/organization/builder.ts

```diff
@@ -4,11 +4,10 @@ import { Builder } from '@commercetools-test-data/core';
 import generator from './generator';
 import transformers from './transformers';
 
-const Organization: TCreateOrganizationBuilder = ({ defaults } = {}) =>
```

Comment:
The types also need to be adjusted.

Reply:
Thanks. Changed. Good we define all types so often ;)